### DISCUSSION TITLE
Welcome to Proxyman Discussions!

### DISCUSSION THREAD

**Discussion by NghiaTranUIT** [ProxymanApp/Proxyman#1865] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jiayouzl** [comment]
The price is too expensive, I hope the price can be cheaper.
After the latest version update, the Chinese language in the settings has changed to English.

**Comment by ghost** [comment]
Currently experiencing cyber attack, apple, google, Amazon, using all platforms to access info and no one entity has a protocol to deal with it. They manipulate all network and traffic. Teaching myself computer science non stop to combat it myself, only 3 months of knowledge under my belt so be kind. Currently intercepting all suspicious requests with a proxy server, just not sure how to respond
